libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.h

```diff
@@ -26,11 +26,12 @@
 #ifndef ESP8266HTTPClient_H_
 #define ESP8266HTTPClient_H_
 
-#include <memory>
 #include <Arduino.h>
 #include <StreamString.h>
 #include <WiFiClient.h>
 
+#include <memory>
```

Comment:
Why is this include needed? There already is a std::unique_ptr in the class (`_payload`), so it seems it's already included.

Reply:
see L29 just above, it was before Arduino.h previously

Reply:
Oh sorry, didn't see that!